week-1-sean-wallace.js

```diff
@@ -0,0 +1,208 @@
+
+
+// --------------------3) What will this log?
+
+const languages = ["JavaScript", "Ruby", "Python", "C++"]
+const index = 1
+console.log(languages[index])
+
+// a) Your answer: Ruby
+// b) Verify and explain:  because it string starts at 0 ruby is in 1.
+
+
+// --------------------4) What will this log?
+
+const weekendDays = ["saturday", "sunday"]
+console.log(weekendDays.toUpperCase())
+
+// a) Your answer: capitalize all caps in weekendDays
+// b) Verify and explain: not gettin a output. but i think it will uppercase all in the weekendDays because of the .toUpperCase console.log.
+
+
+// --------------------5) What will this log?
+
+const dataTypes = ["number", "string", "Boolean"]
+console.log(typeof dataTypes.length)
+
+// a) Your answer: Number
+// b) Verify and explain:Number is datatype this is a type of javascript data type.
+
+
+// # ASSESSMENT 1: Tech Interview Practice Questions
+// Answer the following questions.
+
+// First, without external resources. Challenge yourself to answer from memory as if you were in a job interview. If you don't know the answer a good tip is to answer a simpler version of the question.
+
+// Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+// INSTRUCTOR EXAMPLE: What is a conditional statement?
+
+//   Your answer: the decision a computer makes using a logic. to give an out put by using if else conditional statements.
+
+//   Researched answer: A conditional statement is else if. if condition is true do this. Else, if the condition is false, do something different.
+
+
+
+// 1. What is git? git is open source version control system and is a tool for source code used to devop management for projects.
+//  What is the difference between git and Github?
+
+//   Your answer: github is a website that holds repositories for developers to program and develop ideas.  
+
+//   Researched answer: Git is software for tracking changes in a set of files. github is where developers shape the future of software.
+
+
+
+// 2. Which JavaScript operators will return a Boolean value?
+
+//   Your answer: boolean value represents a true or false. boolean is a form of logic. 
+
+//   Researched answer: the logical operators in javascript &&. True if boolean is true. else then it evaluates false. Logical or true values in boolean results true values, false if both are false, represented by ||. Logical expressions return a true value expressed by true   1 or false 0 value !.
+
+
+
+// 3. What is an index? What is the difference between index and value?
+
+//   Your answer: an index is the indexOfmethod this returns the index; by locating the index of array 
+
+//   Researched answer: The index: Is the index of method and searches from left to right, returns the first position of a specified value and the difference is where the value is stored.
```

Comment:
Index represents the location of a character in a string or element in an array.  The value is the actual content within that location.